### DISCUSSION TITLE
How to switch from old UI to latest version?

### DISCUSSION THREAD

**Discussion by PeteJack912** [aclap-dev/video-downloadhelper#1317] [Help and Q&A]
### Your question

Hi.

I apologise in advance if this is a silly question (my eyesight is limited, so I may have missed something obvious).

I recently clicked on the option to use VDL's old UI. I would now like to change back to the new UI, but can't seem to find how to do so. Can anyone help please?

Thank you.

### Addon & CoApp version and details

Video DownloadHelper
Version 9.1.0.44 (stable)
Browser locale: en-GB
Production build
Built on 2024-09-04
Build options: browser=firefox, linuxlic=false, noyt=false, target=mozilla

Companion app: VdhCoApp 2.0.19

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

With the legacy UI enabled in VDH 9, open the VDH settings the way that has always worked.  On that page, there is a button that did not used to appear in VDH 8 and earlier.  That button restores the new VDH 9 user interface.

![#01](https://github.com/user-attachments/assets/26421bb5-a09e-4045-9633-9d8368534ea5)

**Comment by PeteJack912** [comment]
Ah, I see now that my question was indeed a silly one!

All is well now, thanks to your very clear instructions.

Thank you very much for your assistance. Much appreciated.
